### PR TITLE
added useref hook to handle clicks outside of search bar while it's b…

### PR DESCRIPTION
**Pull request by danielterry2121**
Added a use ref call to close the search field whenever you click away from it, imported the css files.

 [close search results on click away](https://trello.com/c/qfGhh65f/47-close-search-results-on-click-away)

### CODE REVIEW

**Comment by PaulMEdwards** [comment]
[close search results on click away](https://trello.com/c/qfGhh65f/47-close-search-results-on-click-away)

**Review by PaulMEdwards — APPROVED** [review]
Works nicely!

File: src/components/search/SearchField.js [hunk 1]
```diff
@@ -2,18 +2,21 @@ import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import { Button, Modal, Card } from 'antd';
 
+import '../search-bar/Modal.css';
```
Comment (PaulMEdwards) [inline src/components/search/SearchField.js:5]: @PaulMEdwards remember to replace this CSS reference with the LESS version after this PR is merged to main.

**Review by william-jensen01 — APPROVED** [review]
Just make sure to replace the CSS reference like Paul mentioned. Other than that, looks good to me.

**Pull request merged**